src/controllers/addressController.js

```diff
@@ -0,0 +1,22 @@
+const Address = require('../models/Address')
+
+const addressController = {};
+
+addressController.getAddress = async (req, res) => {
+  const { params: { id: addressId } } = req;
+
+  const address = await Address.findOne({ _id: addressId });
+
+  if (!address) {
+    console.log(`No address with ID : ${addressId}`);
```

Comment:
lets think about this. if no address found, what should be returned to the client

Reply:
That is going to be an error catcher, right now I placed only a stub